flytekit/extras/accelerators.py

```diff
@@ -133,7 +133,11 @@ def to_flyte_idl(self) -> tasks_pb2.GPUAccelerator:
 
 #: use this constant to specify that the task should run on an
 #: `NVIDIA L4 Tensor Core GPU <https://www.nvidia.com/en-us/data-center/l4/>`_
-L4 = GPUAccelerator("nvidia-l4-vws")
+L4 = GPUAccelerator("nvidia-l4")
```

Comment:
@mhotan Do you think changing the id can be BC breaking?

GCP is okay with both strings, but is this the same for AWS?

Reply:
AFAIK, we've never tested this gpu accelerator code in aws (we couldn't get those gpus at the time). Looking at the code this should work, but it requires verification.

Reply:
@eapolinario i've used `node_selector={"k8s.amazonaws.com/accelerator": "nvidia-tesla-l4"}` to get l4 to work on aws. we've l4 enabled on the demo hosted tenant.